### PR TITLE
Rename bot(s) to flow(s)

### PR DESCRIPTION
**Pull request by emersonwood**

### CODE REVIEW

File: flows/codelingo/_functions/panel/lingo_bot.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+name: panel
```
Comment (waigani) [inline flows/codelingo/_functions/panel/lingo_bot.yaml]: Change filename to lingo_func.yaml

File: README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ The CodeLingo Hub is an Open source collection of CodeLingo Resources.
 * Lexicons - Libraries for DSLs, such as a language AST or VCS. Provides facts for queries.
 * Tenets -  Patterns and anti-patterns in your software stack written in CLQL.
 * Flows - pipelines for process augmentation and automation.
-* Bots - agents that integrate with your services and data stores.
+* Functions - agents that integrate with your services and data stores.
```
Comment (BlakeMScurr) [inline README.md]: Describing functions as agents seems slightly odd. Perhaps something like "a composable unit of logic to perform a service like interacting with external APIs and datastores."

Comment (waigani) [reply README.md]: I'm rewriting this whole README. It's not worth referencing functions at all at this level.

Comment (BlakeMScurr) [reply README.md]: +1

**Pull request merged**